src/components/Sections/utils.ts

```diff
@@ -89,14 +90,17 @@ export const formatSections = (
   // Handle the case where no layout is provided or layout is an empty array
   // Return folders sorted alphabetically by name, using the default layout
   if (processedLayout.length === 0) {
-    return folders
-      .sort((a, b) => a.name.localeCompare(b.name))
-      .map(folder => ({
-        id: folder.id,
-        name: folder.name,
-        items: folder.items,
-        layout: _defaultLayout
-      }))
+    return [
+      folders
+        .sort((a, b) => a.name.localeCompare(b.name))
+        .map(folder => ({
+          id: folder.id,
+          name: folder.name,
+          items: folder.items,
+          layout: _defaultLayout
+        })),
+      []
+    ]
```

Comment:
Related to previous comment:
For example here, I had to scroll up to search the definition and understand what is the first array and what is the second array. I did not find the answer on the function definition, so I tried to scroll down and found the answer on the last line. Using an object would avoid this gymnastic

Reply:
updated